### DISCUSSION TITLE
issue with horizontal reverse by Trackpad

### DISCUSSION THREAD

**Discussion by Bingqun** [pilotmoon/Scroll-Reverser#160] [General]
Hello, I am appreciate to find the practical application. Thank you.
However I met a problem. I unselect the natural scroll in the system setting. And I want to keep the Trackpad as the natural scroll. So I reversed the Trackpad setting, but the horizontal reverse doesn't work with swiping to switch full screen apps. It is still unnatural direction. The horizontal reverse works in Launchpad but not with full screen switching.
<img width="407" alt="Bildschirm­foto 2022-11-03 um 00 32 48" src="https://user-images.githubusercontent.com/48170674/199622779-6d583f05-c375-4044-b68d-746d90c8b185.png">

**Comment by pilotmoon** [comment]
Scroll Reverser cannot reverse swipes or other gestures. Only scrolling.

I recommend inverting everything: set the system to natural scroll and use Scroll Reverser to reverse the mouse only.
